RELEASE-PROCESS.md

```diff
@@ -8,29 +8,38 @@ these steps when releasing to production:
 
 ## 1. Create a release branch and make a pull request
 
-  - Create a branch for the release called `release-X` where X is the release
+  - Create a branch from `develop` for the release called `release-X` where X is the release
     number
-  - Update [CHANGELOG.md](CHANGELOG.md) to document the changes in this release
+  - Update [CHANGELOG.md](CHANGELOG.md) to:
+     - document the changes in this release in a bullet point form
+     - add a link to the diff at the bottom of the file
+  - `git push <your-branch-name>`, e.g. `git push release-45`
   - Create a tag for the release in the format `release-X`
-  - Create a pull request for the release
+  - Create a pull request for the release with content from the `CHANGELOG.md`
+  - Get that pull request reviewed and approved
 
-## 2. Confirm the release candidate and perform any prerequisites
+## 2. Review and merge the release pull request
+
+The pull request should be reviewed to confirm that the changes in the release
+are safe to ship and that CHANGELOG.md accurately reflects the changes
+included in the release.
+
+## 3. Confirm the release candidate and perform any prerequisites
 
   - Confirm the release with any relevant people (product owner, delivery
     manager, etc)
   - Think about any dependencies that also need considering: dependent parts
     of the service that also need updating; environment variables that need
     changing/adding; third-party services that need to be set up/updated
 
-## 3. Review and merge the release pull request
+## 4. Announce the release
 
-The pull request should be reviewed to confirm that the changes in the release
-are safe to ship and that CHANGELOG.md accurately reflects the changes
-included in the release.
+Let the team know about the release. This is posted in Slack under `#ccs-data-submission`.
+Typical form is:
 
-## 4. Announce the release
+`@here :badger: Release N of <API/Service> going to production :badger:`
 
-Let the team know about the release.
+Acknowledgement is usually made by [`:mushroom:`](https://www.youtube.com/watch?v=6joOVjEemh4)
```

Comment:
I believe it can also be [`:snake:`] randomly as well?

Reply:
PR me